### PR TITLE
remove boardwalk

### LINKED ISSUE #1077: Remove links to Boardwalk from the Terra Data Library
Currently the TOPMed and GTEx datasets in the data library link to two data portals: Windmill Boardwalk. Boardwalk is being deprecated. Remove the link.
screen shot 2019-01-16 at 2 01 02 pm

### PR DESCRIPTION
**Pull request by elizzhao**
Resolves #1077

### CODE REVIEW

File: src/pages/library/Datasets.js [hunk 1]
```diff
@@ -108,22 +108,13 @@ class Participant extends Component {
...
-    tooltip: browseTooltip
-  }, ['Browse Data via Boardwalk'])
-])
+const NIHCommonsButtons = buttonPrimary({
+  style: { margin: '0.25rem 0' },
+  as: 'a',
+  href: 'https://dcp.bionimbus.org/',
+  target: '_blank',
+  tooltip: browseTooltip
+}, ['Browse Data via Windmill'])
```
Comment (panentheos) [inline src/pages/library/Datasets.js:117]: Should this now say 'Browse Data' like the other buttons?

**Review by panentheos — APPROVED** [review]
Looks good.

**Review by breilly2 — CHANGES_REQUESTED** [review]
Also applies to "GTEx presented by NIH Commons". Probably also want to change "Browse Data via Windmill" to simply "Browse Data" like the rest of the buttons... but might want to check with @bradtaylor about that to be sure.

**Comment by elizzhao** [comment]
Keeping "via windmill" after inquiring with Brad. Reasoning: helps indicate people are browsing in a separate system.

**Pull request merged**